xcodeproj/internal/bazel_integration_files/copy_outputs.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+set -euo pipefail
+
+readonly forced_swift_compile_file="$1"
+readonly product_basename="$2"
+readonly exclude_list="$3"
+
+if [[ "$ACTION" == indexbuild ]]; then
+  # Write to "$BAZEL_BUILD_OUTPUT_GROUPS_FILE" to allow next index to catch up
+  echo "i $BAZEL_TARGET_ID" > "$BAZEL_BUILD_OUTPUT_GROUPS_FILE"
+else
+  # Copy product
+  if [[ -n ${BAZEL_OUTPUTS_PRODUCT+x} ]]; then
```

Comment:
Maybe I shouldn't get so cleaver here and just do this?
```suggestion
  if [[ -n "${BAZEL_OUTPUTS_PRODUCT:-}" ]]; then
```

Reply:
👍